src/main/java/com/aws/greengrass/lifecyclemanager/GenericExternalService.java

```diff
@@ -776,10 +775,6 @@ protected Exec addUserGroup(Exec exec, String user, String group) {
      * @return the exec.
      */
     protected Exec addPrivilegedUser(Exec exec) {
-        if (PlatformResolver.isWindows) {
-            logger.atWarn("Windows lifecycle steps cannot run as different users");
-            return exec;
-        }
         String user = Platform.getInstance().getPrivilegedUser();
         String group = Platform.getInstance().getPrivilegedGroup();
         return addUserGroup(exec, user, group);
```

Comment:
Right now windows getPrivilegedUser returns null. `addUserGroup` won't add the group if the user is null